repos.yaml

```diff
@@ -621,3 +621,20 @@ repositories:
     maintainers:
       - email: sre@softonic.com
         name: Softonic
+  - name: aws
+    url: https://aws.github.io/eks-charts
+    maintainers:
+      - email: stefan.prodan@gmail.com
+        name: Stefan Prodan
+      - email: nic@amazon.com
+        name: Nick Turner
+      - email: jsstraub@amazon.com
+        name: James Straub
+      - email: meduri@amazon.com
+        name: Kiran Meduri
+      - email: sabhayav@amazon.com
+        name: Vipul Sabhayav
+      - email: jaypipes@amazon.com
```

Comment:
This can be jaypipes@gmail.com

Reply:
I got this list from Nick and I think it's best to use the Amazon emails. Let me know if this is a blocker and if you want me to change it.

Reply:
Sure, yeah, that's fine. I just recommended my gmail since my Github account is associated with it. But no worries :)

Reply:
You can always change it later